Adafruit_ADT7410.cpp

```diff
@@ -75,16 +122,51 @@ float Adafruit_ADT7410::readTempC() {
   return temp;
 }
 
+/*!
+ *   @brief  Reads the 16-bit temperature register and returns the Fahrenheit
+ *           temperature as a float.
+ *   @return Temperature in Fahrenheit.
+ */
+float Adafruit_ADT7410::readTempF() {
+  uint16_t t = read16(ADT7410_REG__ADT7410_TEMPMSB);
+
+  float temp = (int16_t)t;
+  temp = temp * 0.0140625 /*( 1.0/128.0 * 9.0/5.0 )*/ + 32.0;
+
+  return temp;
+}
```

Comment:
instead of duplicating code, you can call `readTempC()` and then convert that return
```cpp
return readTempC() * 0.0140625 + 32.0;
```

Reply:
careful with that -- you're propagating FP error on the divide in `readTempC()`, and then again on the multiply. also, your `begin()` suggestion above legitimately simplifies maintaining those initialization routines, whereas this function call addition doesn't really gain anything (other than roundoff error!) IMO

Reply:
the adt is way less precise than single floats

Reply:
ah, right you are